queue_management/data/ir_cron.xml

```diff
@@ -4,10 +4,20 @@
         <field name="name">Token: Auto-vacuum log</field>
         <field name="model_id" ref="model_queue_token_location_action" />
         <field name="state">code</field>
-        <field name="active" eval="False" />
```

Comment:
This was on purpose. If we want to remove it, we can enable it directly on the instance, because it is noupdate....

Reply:
done 👍🏽